sdc-scissor.py

```diff
@@ -27,19 +28,22 @@ def cli() -> None:
 @cli.command()
 @click.option('-c', '--count', type=int, default=10)
 @click.option('-d', '--destination', default=_DESTINATION, type=click.Path())
-def generate_tests(count: int, destination: Path) -> None:
+@click.option('-t', '--tool', default='frenetic', type=click.STRING)
+def generate_tests(count: int, destination: Path,tool: str) -> None:
     """
     Generate tests (road specifications) for self-driving cars.
     """
     logging.info('generate_tests')
+    destination = Path(destination)
```

Comment:
Is it possible to instantiate a `Path` object if the path does not exist? I am concerned that this might throw an exception.

Reply:
Yes, we can instantiate if path doesn't exist. 
But I found when parent folder doesn't exist it gives error. So changed to `destination.mkdir(parents=True)` so it's create parent folder also